### ISSUE TITLE
Update Acme.Blog repository to match getting started

### ISSUE THREAD

**Issue by albe** [neos/flow-development-collection#2057]
See https://github.com/neos/Acme.Blog/issues/2

**Comment by mhsdesign** [comment]
I think the real problem is not updating the code 1:1 but that many things are conceptionally outdated.

There is a big contrast between the blog tutorial and fx. @robertlemke 's Flow + React tool Kabana which was presented in the 2022 Neos conference talk "building a modern flow app"

**Comment by albe** [comment]
Of course, a full revamp of the getting started example code would be the best solution. But the lower hanging fruit is at least not having the code in the repository diverge from the the example in the docs :)
